test/spawn-options.spec.js

```diff
@@ -141,6 +141,11 @@ describe('Spawn options', () => {
             expect(_ipfsd).to.exist()
             expect(_ipfsd.api).to.not.exist()
 
+            // proc nodes do not reuse initialized repos
```

Comment:
I think there is no reason why we can't do this in 'proc' nodes as well.

Reply:
Yeah, we even skip this whole describe block for `proc` nodes just above on line 122. My thought was that proc nodes always set their `initialized` to false ([here](https://github.com/ipfs/js-ipfsd-ctl/blob/master/src/ipfsd-in-proc.js#L36)) so once line 122 gets resolved, this would fail for those nodes.